WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -227,6 +227,8 @@ def post_results
     end
     post = Post.new(title: title, body: body, author: current_user, world_readable: true)
     post_post(post)
+    comp.results_posted = Time.now
+    comp.save!
```

Comment:
Since `post_post()` returns the redirect or render, I think it needs to be last. Does this really work in the browser?

I prefer `comp.update(results_posted: Time.now)`

I would also inline `post` above while I was here.